README.md

```diff
@@ -0,0 +1,173 @@
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rails-blog-app <a name="about-the-project"></a>
+
+The **Blog App** is a classic example of a blog website. It is a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/ror">Linters</a></li>
+
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">ROR</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Use database migration files to maintain database schema.
+- Understand ORM
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby on Rails](https://rubyonrails.org/) installed and running. To get more information, read the [installation guide](https://guides.rubyonrails.org/).
+
+- [PostgreSQL](https://www.postgresql.org/) installed and running
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone git@github.com:Gmanlove/Blog-app.git
+  cd Blog-app
+```
+
+### Install
+
+Install the project dependencies with:
+
+```
+  bundle install
+```
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command:
+
+```
+  rails server
+```
+
+### Run tests
+
+To run all tests, including unit and integration tests, run the following command:
+
+```
+  rails test:all
+```
+
+### Deployment
+
+You can deploy this project using:
+
+```
+  Not available yet
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Obi Emmanuel**
+
+- GitHub: [@githubhandle](https://github.com/Gmanlove)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Processing data in models.
+- Views.
+- Forms
+- Add authorization rules.
+- Integration specs for Views.
+- Add API endpoints.
+- API documentation.
+- Validations, Model specs, and n+1 problems.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
```

Comment:
- [OPTIONAL] Great job 👍🏻 But I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```

Reply:
resolved